src/main/java/hudson/plugins/jira/JiraSite.java

```diff
@@ -1172,15 +1081,14 @@ public FormValidation doValidate(@QueryParameter String url,
                 return FormValidation.error( Messages.JiraSite_readTimeoutMinimunValue( "1" ));
             }
 
-            site.setTimeout(timeout);
-            site.setReadTimeout(readTimeout);
             site.setThreadExecutorNumber(threadExecutorNumber);
-            JiraSession session = null;
+
+            session = null;
             try {
                 session = site.getSession();
                 session.getMyPermissions();
                 return FormValidation.ok("Success");
-            } catch (RestClientException e) {
+            } catch (RuntimeException e) {
```

Comment:
that's significant change of method contract, seems not related to described changes

Reply:
This change was done to cover the frequent NPE case and to log it accordingly instead of throwing it as this method is supposed to return FormValidation error, so for me the contract stays the same.